dynparquet/reader.go

```diff
@@ -52,24 +28,30 @@ func (b *SerializedBuffer) Reader() *parquet.Reader {
 }
 
 func (b *SerializedBuffer) ParquetFile() *parquet.File {
-	return b.f
+	f, err := parquet.OpenFile(bytes.NewReader(b.file), int64(len(b.file)))
+	if err != nil {
+		panic("at the disco")
```

Comment:
haha.

Is there no way to return the error though?

Reply:
hah! Thanks, yea I'll replace those with error propagation if this is something we want to merge. Just didn't want to go through all that if this wasn't an approach we liked.

Reply:
Always rather refactor and return errors than panic. If it can be somewhat ignored then maybe logging it is fine. 

Either way:
> [Don't panic.](https://github.com/golang/go/wiki/CodeReviewComments#dont-panic)

Reply:
Panics have been removed